aqt/installer.py

```diff
@@ -676,6 +684,8 @@ def run(self, arg=None):
         return result
 
     def call_installer(self, qt_archives, base_dir, sevenzip, keep):
+        if not qt_archives.get_archives():
+            raise NoPackageFound("Couldn't find any archives")
```

Comment:
I will be raising earlier in `ToolsArchives` with a better error message, but I think it's fair to have this as a fallback...

Reply:
Hum this addition is this resulting in the Azure failures: https://dev.azure.com/miurahr/github/_build/results?buildId=4250&view=logs&j=e9afaa34-1a0f-534e-78fc-fae528ccd915&t=55eb6c4a-c635-53c7-061a-ddccec759601&l=24

```
python -m aqt src 6.1.0 linux desktop --archives qt
```